quarkus-app/src/main/resources/META-INF/resources/css/notifications.css

```
@@ -220,6 +227,11 @@
 }
 
 .notifications-center .btn-link:focus {
+
+/* Ensure inline action buttons in notifications meet 44x44 */
+.notifications-center .notif .actions .btn-link {
+  min-width: 44px; min-height: 44px;
+}
   outline: 2px solid var(--color-accent);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)